build/build.ps1

```diff
@@ -8,14 +8,44 @@ $all_ok = $True
 
 if ($Env:ENABLE_NATIVE -ne "false") {
     Write-Host "##[info]Build Native simulator"
```

Comment:
All this block should go into its own `build-native.ps1` so we can also use it in the e2e build.

Reply:
More specifically, what I would do is break this into two `build.ps1` script that go into the `src\Simulation\Native` and `src\QirRuntime` accordingly, so you can call them separately when working on either project independently.
Then I would create a `build\build-native.ps1` script that gets called by this `build\build.ps1` which the only thing it does is called the other two. That way it will be easier to integrate with the e2e builds.

Reply:
Have split the build script. Should I do the same with the test script as well?

Reply:
moved into build-native-simulator.ps1

---
In reply to: [508211204](https://github.com/microsoft/qsharp-runtime/pull/396#discussion_r508211204) [](ancestors = 508211204)

Reply:
added Simulation\Native\build-native-simulator.ps1 and QirRuntime\build-qir-runtime.ps1

---
In reply to: [508214902](https://github.com/microsoft/qsharp-runtime/pull/396#discussion_r508214902) [](ancestors = 508214902)